### DISCUSSION TITLE
/hyndsight/statistics-vs-econometrics/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#194] [Announcements]
I am not an econometrician 
 https://robjhyndman.com/hyndsight/statistics-vs-econometrics/

**Comment by robjhyndman** [comment]
```header
Originally posted by Letian on 2014-07-21 05:38:23
````
A naive question. If all models are wrong, who do we, especially
econometricians, pay so much attention to asymptotic properties,
biasness and etc? They are wrong as well, right?

**Reply by robjhyndman** [reply]
```header
Originally posted on 2014-07-21 05:44:09
````
Far from naive, that\'s a very good question! Sometimes asymptotic
properties provide a useful first order approximation to small sample
properties, and sometimes bias provides a handle on how useful a model
is under given conditions. But often such research seems to me to be a
little pointless.

**Comment by robjhyndman** [comment]
```header
Originally posted by Miguel A. Arranz on 2014-07-21 11:19:43
````
I partly disagree on several issues. I cannot see much difference
nowadays between a modern econometrician and an applied statistician. I
must admit that I might be biased, since I am econometrician teaching
Econometrics to BSc Statistics students. It is true that there are
different names (we all have funny stories) and that we might use
different subsets of techniques (what you call Survival Analysis is
essential for what we call Duration Models in Microeconometrics, for
example, and GLMs are its core), given the nature of the phenomena we
are analyzing. That means that I concentrate on teaching the techniques
that most statisticians overlook.

However, Econometrics is becoming increasingly data-driven, even
incorporating many concepts from data mining, paying special attention
to visualization of data. It is no longer acceptable to get a (complex)
functional form derived from Economic Theory and test it. Furthermore,
JASA, JRSSB, \... appear on the reading lists of many graduate
econometrics courses, and they are also included in the recommended
lists of publishing journals in many departments of Econometrics.

I think there is still a gap, but is closing much faster than we think.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2014-07-21 11:44:49
````
I hope you\'re right. However, only three days ago I sat through a
seminar involving a relatively complex model and never once saw a plot
of the data. I think there\'s a long way to go before econometricians
get serious about data visualization.

I\'ve updated the article to take account of some of your other
comments. Thanks.

**Reply by robjhyndman** [reply]
```header
Originally posted by econstudent on 2014-07-21 15:16:06
````
There appear to be a generalization error here. :)

I agree that data visualization is not widely taught or appreciated as
it should be in econometrics.

But I don\'t think this is a fair comparison. Econometrics may be better
thought of a particular branch of statistics, in which people develop
and use statistical methods to deal with economic data and test economic
hypotheses. In my mind, this is what defines econometrics. Also
econometricians do look at data. Just think about the concept of
cointegration and ARCH. These econometric concepts would be not possible
if the people hadn\'t looked at the data carefully. But I do get annoyed
with many papers when they don\'t plot their data.

In addition, statistics is a broad topic. For example does every
biostatistician knows a lot about survey analysis, or more than what a
microeconometrician knows about survey analysis?

Last but not least, the concept of hypothesis testing, bias, DGP etc
were not brainchild of econometricians, right?

I am extremely surprised that you have the impression that
econometricians think of their model as being &quot;true.&quot; I can understand
that in some cases you need to assume certain things to make tractable
inference. But it is still an assumption. The readers can feel free to
disagree of course, but it does not mean the author believes it is true.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2014-07-21 23:41:31
````
I agree on cointegration and ARCH \-- brilliant ideas proposed by some
excellent econometricians. I knew Clive Granger and he certainly looked
very carefully at any data he analysed.

However, I never said that econometricians needed to know all areas of
statistics. Read what I wrote more carefully. I was simply reflecting on
the differences I have observed, and how statisticians are trained more
broadly. There are good reasons for that as you have suggested.

I also never said that hypothesis testing or bias calculations were bad.
My point was that they should not come before a careful look at the
data. (This is anathema in some areas of social science.)

I\'ve no idea who came up with the phrase DGP. I never heard it before I
moved to an econometrics department, and I imagined it was a peculiarity
to econometrics. Perhaps it is used in statistics as well, but not by
the statisticians I know.

**Reply by robjhyndman** [reply]
```header
Originally posted by Capistran on 2014-07-27 21:10:56
````
Clive Granger was also a statistician that spend a lot of time in an
econometrics department. Looking at the data is of course fundamental,
and econometricians have to do it much more. Although I have to say that
as an econometrician, I have also had the opposite experience: looking
at too many charts and too little theory!

**Comment by robjhyndman** [comment]
```header
Originally posted by Chris on 2014-07-21 22:10:12
````
I worked in a Health Economics department in a large &quot;Global &quot; Pharma
company. Before that I had worked strictly on phase I, II, III clinical
trials and a drug approval or two. I spent a lot of time explaining
economists/econometrics to the statisticians and explaining
statistics/experimental design to the economists. I still don\'t think
they understand the hypothesis of non-inferiority. My running joke was
&quot;\...dummy variables are for dummy\'s.,,..&quot; and theirs &quot;\... we
don\'t need no stinking confidence intervals\...&quot;. We did get along
extremely well, mainly because I actually did know how to analyze a
double blind randomized clinical trial - until a corporate restructuring
and downsizing. We remain good friends, and I even got a couple sole
author publications out of the work.

**Comment by robjhyndman** [comment]
```header
Originally posted by Martha Smith on 2014-07-22 03:10:18
````
An interesting compare-and-contrast. Thanks.\
I have noticed that the &quot;theory, then collect data, then test
hypothesis \-- but no reason to plot the data&quot; paradigm also seems to
be prevalent in psychology.\
But I am used to using &quot;robust&quot; in more or less the way you say
econometricians do: &quot;robust&quot; meaning insensitive to violations of
model assumptions, and &quot;resistant&quot; meaning insensitivity to outliers.\
(I am a mathematician, but got into teaching graduate statistics courses
a number of years ago, with greatest interest in applications in biology
and engineering. I am definitely not an econometrician!)

**Comment by robjhyndman** [comment]
```header
Originally posted by Vincent Granville on 2014-07-22 14:30:49
````
Can\'t you be both? I thought econometricians were statisticians working
with time series models, ARMA processes etc.

**Reply by robjhyndman** [reply]
```header
Originally posted by Stefan Janse van Rensburg on 2014-07-24 10:04:55
````
Vincent, that sort of describes me\... Due to weird subject choices at
university, I ended up taking econometrics type subjects (time series
etc) with a stats department at the expense of the traditional stats
topics like experimental design and GLM\... But I didn\'t quite do
enough economics, so I feel a little like a half-qualified statistician
and a half-qualified econometrician\... So call me an empirical
econometrician :-P Though I\'m not sure that such a creature is the most
useful thing.

**Comment by robjhyndman** [comment]
```header
Originally posted by Robert Taylor on 2014-07-22 19:05:05
````
Well now. I was trained as a statistician but work as an econometrician.
The latter don\'t like to look at data because they fear that doing so
will invalidate all their asymptotic theory (pre-test bias etc). I was
taught always to look at the data and see what it\'s telling you. To me
that seems sensible. But the econometrics profession is grinding it out
of me :)

**Comment by robjhyndman** [comment]
```header
Originally posted by Eric on 2014-07-31 02:39:23
````
I\'m not sure how you can do good econometrics without a thorough
background in statistics. I\'m also dismayed to learn that people run
models without thoroughly studying their data. On the other hand, it\'s
been 20 years since I gave up my academic appointment for Wall St. so
maybe things have changed. On the applied side, I see a ton of studies
that use reduced form models. even if the authors don\'t know that this
is what they are doing \-- so as an empirical matter I don\'t buy the
idea that econometrics is more concerned with structure (whether
economic structure or data structure). Perhaps the distinction is that
good, rigorous econometrics requires statistics and often structure, but
bad econometric studies can do without.

**Comment by robjhyndman** [comment]
```header
Originally posted by lu-nonymous on 2015-01-28 08:59:18
````
Hi, I\'m a PhD student doing research in empirical macroeconomics. Your
last sentence got me to respond. I\'m actually very much interested in
data visualization and most of the data I stare at is macro data. I
think we can benefit very strongly from better data visualization.

On problem with macro data is that it\'s &quot;small&quot;. You wouldn\'t want
to visualize US postwar annual GDP, consumption and investment time
series for your seminar audience, because they\'ve all seen it countless
times.

People might be more interested in it, if it\'s historical macro data
(e.g. Piketty). Or if you\'ve found a way to get data on a higher
frequency for more countries.

**Comment by robjhyndman** [comment]
```header
Originally posted by 10is Maestro on 2016-12-28 05:07:26
````
It is a dated discussion, but, if you want an update, I just completed
an undergraduate degree in economics \-- with several econometrics
course, including a graduate course in econometrics.

If you want an example of data driven choice, consider fitting a moving
average model, an autoregressive model or a combination of both on a
given time series. A quick ad hoc check you can do is look at
correlograms (both partial and normal) knowing that the impulse response
function of all three processes differ greatly \-- e.g., MA process have
a shorter memory whereas shocks reverbate for very long in AR processes.
This obviously have implications for modeling from a statistical
perspective, but also as far as economic theory is concerned: if you can
well approximate the behavior of a time series with a moving average
model, a theory that gets you long lasting effects and slow adjustment
likely is wrong. So, correlograms is one way we often see what is going
with macro and financial data.

Another usual plot we perform is a simple line plot of the time series
\-- does it look stationary? If it is too messy, we should bother to use
unit root tests and make sure we work with stationary series. This is
actually something we always do \-- and if I pick a growth rate or a
first difference, I also plot it to see if I get what I expect. Those
plots can also reveal heteroskedasticity \-- perhaps episodes of high
and low variance, which may be well represented by ARCH processes.

We also usually pick ranges of lags with correlograms and formalize that
argument with information criteria \-- which also are data driven.

Remember that it is not because you do not see visual representations in
seminars that no one used it. They might simply believe it is not
sufficiently forceful or superfluous.

**Comment by robjhyndman** [comment]
```header
Originally posted by Marius Ooms on 2017-05-22 10:09:07
````
Hi Rob,

At the
[VUamsterdam.com](http://VUamsterdam.com &quot;VUamsterdam.com&quot;)
we try to close the gap with a new Bachelor &quot;Econometrics and Data
Science&quot; with contributions from the Economics and the Science
department. In May 2017 we have 100 applicants so far. In the Masters,
we hope to join Time Series Analysis and Data Science. Amazon is a key
employer in the world. In the Netherlands it is
[Booking.com](http://Booking.com &quot;Booking.com&quot;)
with plenty of internships, supervised by former econometric time series
students.

Best,\
Marius.

**Reply by robjhyndman** [reply]
```header
Originally posted on 2017-05-22 23:03:16
````
That sounds very progressive! I visited the
[Booking.com](http://Booking.com &quot;Booking.com&quot;)
data science team in Amsterdam last year. They are doing some great
work.

**Comment by robjhyndman** [comment]
```header
Originally posted by Michael Wulfsohn on 2018-05-15 16:09:47
````
Are there any other fields like econometrics that overlap closely with
statistics? Perhaps machine learning? Also, it seems one could make a
career of transferring innovations between the fields.

**Comment by robjhyndman** [comment]
```header
Originally posted by Peter Westfall on 2018-11-15 13:45:01
````
A slight quibble: A chosen (a priori) statistical model is indeed a data
generating process. The extent to which it generates data that resemble
data produced by the real data generating process measures validity of
the chosen model.

**Comment by NWabiri** [comment]
Thanks Rob, Nice article on “I am not an econometrician”; I have had, on several occasions, told people the same, and they seem surprised. I work with economists and coming from systems thinking perspective, the idea of being married to a model and trying to prove it true is fascinating.

**Comment by spengler1918** [comment]
I heartily agree with your sentiments overall. 

But I'm confused about your objection to the phrase "data generating process." Isn't that just a synonym for "statistical model," as the statisticians say? But to me, DGP is more descriptive and direct.

**Reply by robjhyndman** [reply]
"Data generating process" literally means the process by which the data were generated. But the data weren't generated from that process (except in a simulation).
